static/js/stream_edit.js

```diff
@@ -40,6 +40,14 @@ exports.get_email_of_subscribers = function (subscribers) {
     });
 };
 
+exports.get_human_subscribers_count = function (subscribers) {
```

Comment:
If there is really no pre-existing way to get the _human_ subscriber count, this function would be more appropriately placed in `stream_data.js` next to `get_subscriber_count`, and be called `get_human_subscriber_count` (just dropping the `s` in the name).